galvan/src/main/java/org/terracotta/testing/master/ServerProcess.java

```diff
@@ -251,7 +251,7 @@ private SimpleEventingStream buildEventingStream(VerboseOutputStream stdout) {
     Map<String, String> eventMap = new HashMap<String, String>();
     eventMap.put("PID is", pidEventName);
     eventMap.put("Terracotta Server instance has started up as ACTIVE node", activeReadyName);
-    eventMap.put("console - Moved to State[ PASSIVE-STANDBY ]", passiveReadyName);
+    eventMap.put("Moved to State[ PASSIVE-STANDBY ]", passiveReadyName);
```

Comment:
The log statement itself is not specific to a target server. From what I saw that's not even required as the log statements eventbuses are different for each server. So even though the log statement is the same it will be processed separately by each server.